src/ResourceManager/Sql/LegacySdk/SqlManagement.Netcore.csproj

```diff
@@ -1,6 +1,9 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
-  <Import Project="$(ProjectDir)..\..\..\..\tools\Common.Netcore.Dependencies.targets" />
+  <PropertyGroup>
+    <OmitJsonPackage>true</OmitJsonPackage>
+  </PropertyGroup>
+  
```

Comment:
Because this is a captured Hyak library - it doesn't require 90% of the dependencies in the common targets, just the runtime modules related to Hyak

Reply:
there's no reason to use OmitJsonPackage here if we aren't using the common targets, however.

Reply:
I think I was stripping this to bare bones (because it is a captured Hyak library) to try to debug an issue and didn't put it back.  Will try using the common pacjage and letting the compiler decide on dependencies.

Reply:
It just makes it easier updating those packages to not have one-off files.